IBAnimatable/MaskDesignable.swift

```diff
@@ -191,7 +196,13 @@ public extension MaskDesignable where Self: UIView {
   
   private func maskWaveFromString(mask: String) {
     let params = retrieveMaskParameters(mask, maskName: MaskType.Wave.rawValue).componentsSeparatedByString(",")
-    if let unwrappedWidth = Float(params[1]), unwrappedOffset = Float(params[2]) where params.count == 3 {
+    
+    guard params.count == 3 else {
```

Comment:
Not sure that is necessary, but shouldn't we able to use the wave with 1 or 2 parameters if we only need to set the direction... ? Anyway, that can be another PR.